src/enhetsportefolje/enhet-listehode.tsx

```diff
@@ -153,6 +153,30 @@ function EnhetListehode({
                     skalVises={valgteKolonner.includes(Kolonne.TOLKEBEHOV_SIST_OPPDATERT)}
                 />
 
+                <SorteringHeader
+                    sortering={Sorteringsfelt.CV_KAN_DELES_STATUS}
+                    onClick={sorteringOnClick}
+                    rekkefolge={sorteringsrekkefolge}
+                    erValgt={sorteringsfelt === Sorteringsfelt.CV_KAN_DELES_STATUS}
+                    tekst="CV kan deles"
+                    className="col col-xs-2"
+                    skalVises={valgteKolonner.includes(Kolonne.CV_KAN_DELES_STATUS)}
+                    title="CV kan deles"
+                    headerId="cv-kan-deles-status"
+                />
+
+                <SorteringHeader
+                    sortering={Sorteringsfelt.CV_SVARFRIST}
+                    onClick={sorteringOnClick}
+                    rekkefolge={sorteringsrekkefolge}
+                    erValgt={sorteringsfelt === Sorteringsfelt.CV_SVARFRIST}
+                    tekst="CV svarfrist"
+                    className="col col-xs-2"
+                    skalVises={valgteKolonner.includes(Kolonne.CV_SVARFRIST)}
+                    title="CV kan deles"
+                    headerId="cv-svarfrist"
+                />
+
```

Comment:
Det som står her måtte jeg duplisere i minoversikt-listehode.tsx, det samme gjelder litt under her, at jeg måtte skrive nøyaktig det samme for enhet-* og minoversikt-*. Jeg bare lurer litt på hvorfor det er slik, og om det er noe vi tenker å gjøre noe med...? 🤔

Reply:
Eg kjenner ikkje heilt historikken men antakeligvis har det gitt meining å separere desse fordi ein har litt ulik håndtering for nokre ting. Så har det sansynlegvis vokst over tid og ein har endt opp med å duplisere mykje kode/markup mellom desse to filene for kolonner der håndtering er lik. 

Det er fullt mogleg å trekke saman det som er felles og ein kan sikkert argumentere for at dette er ein fin ting. På den andre sida så er det kanskje litt meir forutsigbart å ha dette separert sidan alternativet blir at ein må ha to kontekstar i hovudet samstundes ("Min oversikt" og "Enhetens oversikt") når ein skal gjere ei endring.

Alternativt kan ein sjå på kvar enkelt fil og moglegheita for å gjere det meir kompakt. Her ville det feks. vore mogleg å representere dei ulike kolonnene vha. eit JavaScript dataobjekt, for så å gå gjennom alle kolonnetyper i objektet og mappe til JSX-representasjonen vi ønsker. Vi kan bruke JavaScript rett i JSX, så en mulig løsning kunne feks. sett slik ut:

```jsx
const kolonner: {
    [key: string]: Kolonne
} = {
    etternavn: {
        sorteringsfelt: Sorteringsfelt.ETTERNAVN,
        headerId: "etternavn",
        tekst: "Etternavn",
        tittel: "Etternavn"
    },
    fodselsnummer: {
        sorteringsfelt: Sorteringsfelt.FODSELSNUMMER,
        headerId: "fodselsnummer",
        tekst: "Fødselsnr.",
        tittel: "Fødselsnummer"
    }
}

return (
    <div className='brukerliste__header brukerliste__sorteringheader'>
        <div className='brukerliste__gutter-left' />
        <div className='brukerliste__innhold' data-testid='brukerliste_innhold'>
            <VelgalleCheckboks className='velgalle-checkboks' />
            {Object.values(kolonner).map((kolonne: Kolonne) =>
                <SorteringHeader
                    className="col col-xs-2"    // <-- Forenkling, siden "Etternavn" og "Fødselsnummer" i utgangspunktet hadde ulike CSS-klasser
                    sortering={kolonne.sorteringsfelt}
                    onClick={sorteringOnClick}
                    rekkefolge={sorteringsrekkefolge}
                    erValgt={sorteringsfelt === kolonne.sorteringsfelt}
                    tekst={kolonne.tekst}
                    title={kolonne.tittel}
                    headerId={kolonne.headerId}
                />
            )}
        </div>
        <div className='brukerliste__gutter-right' />
    </div>
);
```

Reply:
Det er skrudd av i prod for nå altså